packages/compass-schema/src/components/field/field.jsx

```diff
@@ -127,7 +127,7 @@ class Field extends Component {
   getSemanticType(type) {
     // check if the type represents geo coordinates, if privacy settings allow
 
-    if (this.enableMaps) {
+    if (this.state.enableMaps) {
```

Comment:
👀

Reply:
Let’s add this to the long list of “TypeScript would have caught this” :smile:

Reply:
💯